### PR TITLE
Delete employees' email detail for future update in emergency contact

### LINKED ISSUE #47: As an intermidiate user, I can edit my employees' contact details
... so I can keep the latest updates.

### PR DESCRIPTION
**Pull request by Persdre**
For a future update in emergency contact

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -39,12 +37,11 @@
...
             + "[" + PREFIX_NAME + "NAME] "
             + "[" + PREFIX_PHONE + "PHONE] "
-            + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_PHONE + "91234567 "
-            + PREFIX_EMAIL + "johndoe@example.com";
+            + PREFIX_PHONE + "91234567 ";
+    // for edit phone's issues, change p/phone to e/emergency_contact or keep it
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: I think we should keep both. They will both be Phone objects, but with different prefixes.

**Review by ureshiiYing — DISMISSED** [review]
Overall, LGTM. I don't think you need to remove the email information under tutorial documentation. However, this does not affect the app, so perhaps it is fine.

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (Persdre) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: Yes, thank you!

Comment (yanbingtao) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: Yup, should keep both. "e/" for emergency contact, "p/" for employee's contact number.

**Review by yanbingtao — APPROVED** [review]
LGTM

**Review by ureshiiYing — APPROVED** [review]
LGTM.

**Pull request merged**